### PR TITLE
Show operatives on work orders on date that appointment has started

### PR DESCRIPTION
**Pull request by EGiataganas**
### Description of change

Show operatives on work orders on date that appointment has started, before we were debating on the current dateTime, where now only on the date

### CODE REVIEW

File: src/models/workOrder.test.js [hunk 1]
```diff
@@ -83,10 +83,10 @@ describe('WorkOrder', () => {
 
       MockDate.set(addMinutes(now, 1))
 
-      expect(workOrder.appointmentStartTimePassed()).toBe(true)
+      expect(workOrder.appointmentISODatePassed()).toBe(true)
     })
 
-    it('returns false when the current time is before the appointment start time', () => {
+    it('returns true when the current time is before the appointment start time', () => {
```
Comment (pixeltrix) [inline src/models/workOrder.test.js:89]: Should we have a test for it being false on a before date?

File: src/models/workOrder.test.js [hunk 2]
```diff
@@ -96,7 +96,20 @@ describe('WorkOrder', () => {
...
+      const now = new Date()
+
+      const date = format(now, 'yyyy-MM-dd')
+      const start = format(now, 'HH:mm')
+
+      const workOrder = new WorkOrder({ appointment: { date, start } })
+
+      MockDate.set(subDays(now, 1))
+
+      expect(workOrder.appointmentISODatePassed()).toBe(false)
```
Comment (siapankina) [inline src/models/workOrder.test.js:112]: this is duplication of the test above 
could you delete it please ?

File: src/models/workOrder.test.js [hunk 3]
```diff
@@ -96,7 +96,20 @@ describe('WorkOrder', () => {
 
       MockDate.set(subMinutes(now, 1))
 
-      expect(workOrder.appointmentStartTimePassed()).toBe(false)
+      expect(workOrder.appointmentISODatePassed()).toBe(true)
+    })
+
+    it('returns true when the current time is before the appointment start time', () => {
```
Comment (siapankina) [inline src/models/workOrder.test.js:102]: since we do, `subDays`
maybe it should be " it returns true when the current day is before the app day"?

Comment (pixeltrix) [inline src/models/workOrder.test.js:102]: Should this be `returns false when the current date is before the appointment date` ?

**Review by pixeltrix — APPROVED** [review]

**Pull request merged**